internal/configuration/schema/totp.go

```diff
@@ -3,4 +3,6 @@ package schema
 // TOTPConfiguration represents the configuration related to TOTP options.
 type TOTPConfiguration struct {
 	Issuer string `mapstructure:"issuer"`
+	Period int    `mapstructure:"period"`
+	Skew   *int   `mapstructure:"skew"`
```

Comment:
Why use a pointer here?

Reply:
So that users can set a skew of 0 which is valid. If we don't use a pointer we can't determine if it was set or not.

Reply:
OK makes sense.